src/commands/dev/lib/dev-server.ts

```diff
@@ -393,23 +392,29 @@ export default class DevServer {
 
     // Retrieve the path of the native module
     const modulePath = await getModuleForNSFW(this.output);
-
-    // Collect files to watch
-    this.files = await collectProjectFiles('**', this.cwd);
-
-    // Start the filesystem watcher
-    this.nsfw = await nsfw(this.cwd, this.handleFilesystemEvents.bind(this), {
-      modulePath
-    });
-
     const [env, buildEnv] = await Promise.all([
       this.getLocalEnv('.env'),
       this.getLocalEnv('.env.build')
     ]);
     this.env = env;
     this.buildEnv = buildEnv;
-
     const nowJson = await this.getNowJson();
+
+    const opts = { output: this.output, isBuilds: true };
+    const files =  await getFiles(this.cwd, nowJson, opts);
+    const results: { [filePath: string]: FileFsRef } = {};
+    for (const fsPath of files) {
+      const path = relative(this.cwd, fsPath);
+      const mode = 33188; // TODO: get the real mode
```

Comment:
This needs to be fixed. Note that the previous implementation leveraged the `statsCache` object that `glob` populates as an optimization.

Reply:
@TooTallNate Ok I'll look into exposing the `statsCache` somehow from `staticFiles` method.

Reply:
@TooTallNate I cheated and just use `stat` again here so we can ship it :shipit: